krave/ui/analized_data/real_time_analized_data.csv

```diff
@@ -0,0 +1,8 @@
+trial,bg_repeat,wait_time,miss_trial
+0,1,0.5643043518066406,False

```

Comment:
is this file supposed to be here permanently? If this is a temporary file, maybe we could add it to .gitignore?